### ISSUE TITLE
ChoiceGroup: WCAG issues

### ISSUE THREAD

**Issue by kristiina-richards** [TEDI-Design-System/react#346]
### Description of the problem

JAWS reads out checkbox and radio button labels incorrectly - first the checkbox and the the label. Should be read only once not as 2 separate items.  
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)
